flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/slotmanager/SlotManager.java

```diff
@@ -205,46 +177,64 @@ public void handleSlotRequestFailedAtTaskManager(final SlotRequest originalReque
 		LOG.info("Slot request failed at TaskManager, SlotID:{}, AllocationID:{}, JobID:{}",
```

Comment:
Can't add comment to the line of function declaration
handleSlotRequestFailedAtTaskManager can be private